src/Struct/FindologicProduct.php

```diff
@@ -358,10 +358,13 @@ protected function setKeywords(): void
     {
         $blackListedKeywords = [
             $this->product->getProductNumber(),
```

Comment:
Is the product number also nullable?

Reply:
No, it is not:
![image](https://user-images.githubusercontent.com/51320851/157005839-a915d002-df27-4473-88a7-5eeaee743a38.png)